browser/src/control/Control.Sidebar.js

```diff
@@ -152,7 +152,7 @@ L.Control.Sidebar = L.Control.extend({
 	},
 
 	changeDeck: function(unoCommand) {
-		if (unoCommand !== null)
+		if (unoCommand !== null&& unoCommand !== '.uno:ModifyPage')
```

Comment:
I don't think we should add a condition here. Instead for the top right button, we can add fix here: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.TopToolbar.js#L242

Reply:
yes I forget to remove this . thanks :)